assets/src/ba_data/python/bastd/actor/spazbot.py

```diff
@@ -324,7 +327,9 @@ def update_ai(self) -> None:
                      target_vel * dist_raw * 0.3 * self._lead_amount)
 
         diff = (target_pt - our_pos)
-        dist = diff.length()
+        dist = distance(
+            (target_pt.x, target_pt.y, target_pt.z),
+            (our_pos.x, our_pos.y, our_pos.z))
```

Comment:
Same here; old code should work now that ba.Vec3.length() is fixed. Please holler if not.